plugins/event_notification/providers/kafka/lib/model/KafkaNotificationTemplate.php

```diff
@@ -114,7 +114,7 @@ public function dispatch(kScope $scope)
 		}
 
 		$requiredPermissions = explode(",", $this->getRequiresPermissions());
-		if(count($requiredPermissions))
+		if(count(array_filter($requiredPermissions)))
```

Comment:
If the purpose is to remove all empty values that return true on the empty() function please state the function when calling array_filter. To avoid case where this logic will change in the future.

From PHP doc:
callback
The callback function to use

If no callback is supplied, all empty entries of array will be removed. See [empty()](https://www.php.net/manual/en/function.empty.php) for how PHP defines empty in this case.